src/server/pfs/drive/driver.go

```diff
@@ -947,7 +947,7 @@ func (d *driver) inspectFile(file *pfs.File, filterShard *pfs.Shard, shard uint6
 	for commit != nil && (from == nil || commit.ID != from.ID) {
 		diffInfo, ok := d.diffs.get(client.NewDiff(commit.Repo.Name, commit.ID, shard))
 		if !ok {
-			return nil, nil, fmt.Errorf("diff %s/%s/%d not found", commit.Repo.Name, commit.ID, shard)
+			return nil, nil, pfsserver.ErrCommitNotFound
```

Comment:
So the info we're losing is repo / commit ID / shard. When invoking GetFile() you provide the repo and commit ID, so it seemed redundant to provide this in the error message. Presumably the caller has enough information to debug.

As long as the other places we return 'commit not found' also have those inputs, I think it's fine to omit. 

But - is the shard number meaningful to report? Maybe for debugging? For the CLI, I'd say no. For the error reported to the client, I think its fine to include.

I will go through the other locations we want to return 'commit not found' and use the canonical err.